src/Spago/Messages.hs

```diff
@@ -146,6 +146,18 @@ upgradingPackageSet newTag = makeMessage
   , "Fetching the new one and generating hashes.. (this might take some time)"
   ]
 
+changingToSpecificPackageSet :: Text -> Text
+changingToSpecificPackageSet newTag = makeMessage
+  [ "Current tag is different from specified tag " <> surroundQuote newTag
+  , "Fetching the new one and generating hashes.. (this might take some time)"
+  ]
+
+nonExistentPackageSet :: Text -> Text -> Text -> Text -> Text
+nonExistentPackageSet org repo oldTag newTag = makeMessage
+  [ "Package-set tag " <> surroundQuote newTag <> " in the repo " <> surroundQuote (org <> "/" <> repo) <> " does not exist."
+  , "Reverting change back to " <> surroundQuote oldTag
```

Comment:
Technically we're not reverting anything here, we're just "not updating the file". The wording with "reverting" might be a bit scary, so I'd like something softer here

Reply:
Good point. I've updated it to a different message now (will post soon).